src/pykiso/lib/connectors/cc_pcan_can/cc_pcan_can.py

```diff
@@ -34,6 +34,16 @@
 except ImportError as e:
     raise ImportError(f"{e.name} dependency missing, consider installing pykiso with 'pip install pykiso[can]'")
 
+try:
+    # use to avoid timestamp inconsistances between pykiso and python can
+    import uptime
+
+    if uptime.boottime() is None:
+        boottime_epoch = 0
+    else:
+        boottime_epoch = uptime.boottime().timestamp()
+except ImportError:
+    boottime_epoch = 0
```

Comment:
```suggestion
boottime_epoch = 0
try:
    # use to avoid timestamp inconsistances between pykiso and python can in linux
    import uptime
    if not uptime.boottime():
        boottime_epoch = uptime.boottime().timestamp()
except:
    pass
```

Reply:
I personally find it more explicit to set it to 0 in all failing cases

Reply:
I also like the fact that we don't catch all errors